requirements/default.txt

```diff
@@ -1,3 +1,4 @@
 celery>=5.2.3,<6.0
 django-timezone-field>=4.2.3
 python-crontab>=2.3.4
+tzdata>=2022.1
```

Comment:
django 4.0 switched to zoneinfo that might be the cause, does adding this package make the docker build happy?

Reply:
zoneinfo requires timezone data either installed as a os package in which case isn't installed as a os package currently via the dockerfiles nor is it part of the base image of python:3.8-slim  or by installing the tzdata python package.   Documentation below.  

https://docs.python.org/3/library/zoneinfo.html#data-sources

`[](https://docs.python.org/3/library/zoneinfo.html#data-sources)Data sources
The zoneinfo module does not directly provide time zone data, and instead pulls time zone information from the system time zone database or the first-party PyPI package [tzdata](https://pypi.org/project/tzdata/), if available. Some systems, including notably Windows systems, do not have an IANA database available, and so for projects targeting cross-platform compatibility that require time zone data, it is recommended to declare a dependency on tzdata. If neither system data nor tzdata are available, all calls to [ZoneInfo](https://docs.python.org/3/library/zoneinfo.html#zoneinfo.ZoneInfo) will raise [ZoneInfoNotFoundError](https://docs.python.org/3/library/zoneinfo.html#zoneinfo.ZoneInfoNotFoundError).`